.gitignore

```diff
@@ -42,3 +42,5 @@ local.properties
 # Sandbox stuff
 _sandbox
 
+signing.properties
+droidconit.jks
```

Comment:
@tiwiz for the keystore, put it in a folder, then inside that folder put a .gitignore that ignores everything (just a `*`), so also itself... :sparkles: magic :sparkles: